samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/bean/author/AuthorBean.java

```diff
@@ -219,6 +227,28 @@ public List getPublishedAssessments(){
 
   public void setAllAssessments(List allAssessments) {
     this.allAssessments = allAssessments;
+
```

Comment:
Is this logic and objects required if the groupFilterEnabled property is disabled?

Reply:
You're right @mpellicer , is not necessary to loop the assessment list at this point if groupFilterEnabled is false, because group variable is only used when groupFilter is enabled.

I've made a rebase on the PR.

Reply:
Thanks!